tests/escrow/tests/escrow.ts

```diff
@@ -115,10 +131,14 @@ describe("escrow", () => {
     assert.ok(_escrowAccount.initializerAmount.toNumber() == initializerAmount);
     assert.ok(_escrowAccount.takerAmount.toNumber() == takerAmount);
     assert.ok(
-      _escrowAccount.initializerDepositTokenAccount.equals(initializerTokenAccountA)
+      _escrowAccount.initializerDepositTokenAccount.equals(
```

Comment:
These account fields don't typecheck properly.

Reply:
If you import from ../../../ts/src instead it properly typechecks

Reply:
oh if you run his generate types script again (createIDLType) and do the thing I outlined above it typechecks correctly. You will see that since you didn't define the rent sysvar is going to show you need to remove it.

Reply:
The above works. :+1: